Code/Framework/AzCore/Tests/Math/Matrix3x4Tests.cpp

```diff
@@ -1010,6 +1010,14 @@ namespace UnitTest
         EXPECT_NEAR(matrix2.GetTranspose3x3().GetDeterminant3x3(), expected2, 1e-3f);
     }
 
+    // Use of INFINITY in newer Windows SDKs trigger a math overflow warning becuase it redefines INFINITY
```

Comment:
typo in because and maybe add the msvc version which introduced  this change

Reply:
I'll fix the typo, but I don't know the exact change in the windows SDK (its an Windows SDK issue, not a msvc compiler issue).    How would I find this out, besides exhaustively diffing every SDK line? in every version?  Its not open source, so there's no github I can find for it to do a diff or blame on.